### ISSUE TITLE
Supported aliases for `"electrical"`.

### ISSUE THREAD

**Issue by 1uc** [BlueBrain/libsonata#229]
When reading the specification:
https://sonata-extension.readthedocs.io/en/latest/sonata_config.html#id5
we see that the supported types are:
* chemical
* electrical
* synapse_astrocyte
* endfoot
* neuromodulatory

However, the SONATA documentation usecase5 contains a type `"electrical_synapse"`, see:

    /gpfs/bbp.cscs.ch/project/proj12/example_data/sonata/v2/usecase5/

Using the following reproducer, we can read in the file on BB5 and print the type which is `"electrical_synapse"`.
```
import libsonata

usecase5_conf = "/gpfs/bbp.cscs.ch/project/proj12/example_data/sonata/v2/usecase5/circuit_sonata.json"

circuit = libsonata.CircuitConfig.from_file(usecase5_conf)
print(circuit.edge_population("astrocyteA__astrocyteA__electrical_synapse"))
print(circuit.edge_population_properties("astrocyteA__astrocyteA__electrical_synapse").type)
```

Please consider either updating the specifications document or make `libsonata` throw on invalid "type".

**Comment by joni-herttuainen** [comment]
Hi @1uc ,

Thanks for the issue. I thought I'd comment on this. 

The examples are generated programmatically. You are right, `"electrical_synapse"` incorrect. It should be `"electrical"`. I have already made a MR for the generator code regarding the matter. IIRC, I held it back as I was not sure whether it conformed to the SimulationConfig specifications and thought I'd do everything in one go. Due to priorities, however, this was pushed back and TBH, I forgot about it completely. I'll add this to my to-do list to make sure I won't forget it again.

Regarding `libsonata`, I agree, `"electrical_synapse"` should throw if that's the expected behavior on incorrect types.

**Comment by 1uc** [comment]
Thank you for the clear explanation.

**Comment by mgeplf** [comment]
"electrical" is the correct type; so let's use that - see #237 and the fix in #242 

The proj12 example data should probably be updated.

I'm going to close this ticket, since it's a duplicate of #237
